openeo/extra/job_management.py

```diff
@@ -0,0 +1,379 @@
+import collections
+import contextlib
+import datetime
+import json
+import logging
+import os
+import time
+from pathlib import Path
+from typing import Callable, Union, Dict, Optional
+
+import geopandas as gpd
```

Comment:
geopandas is not a listed dependency of the client

Reply:
Followed the advice of @jdries to use shapely for converting the geometry column. So we no longer need geopandas as a installation dependency (It was part of the testing dependencies though)